### ISSUE TITLE
Sequence Labeling Task:why does the fine-tuned model evaluated same result as the pre-trained model evaluating?

### ISSUE THREAD

**Issue by croari** [microsoft/unilm#302]
**Describe**
Model I am using  LayoutLM :
I use some my own dataset to fine-tuning the Sequence Labeling Task, the process was normal and completed well. 
And the loss graph that use tensorboard looks convergency.
But now I found that to the same data ,the evaluated result from the fine-tuned model  is same as the pre-trained model evaluated. Sames like the fine-tuning  model had not been used?
Did you have this problem?
Did I do something wrong?

**Comment by sumeetsuman83** [comment]
@croari Did you find any solution for this?
